src/test/scala/uk/gov/nationalarchives/tdr/api/utils/TestUtils.scala

```diff
@@ -111,9 +112,32 @@ object TestUtils {
   def addAntivirusMetadata(fileId: String): Unit = {
     val sql = s"insert into AVMetadata (FileId, Result, Datetime) VALUES (?, ?, ?)"
     val ps: PreparedStatement = DbConnection.db.source.createConnection().prepareStatement(sql)
-    ps.setString(1, s"${fileId}")
+    ps.setString(1, fileId)
     ps.setString(2, "Result of AVMetadata processing")
     ps.setTimestamp(3, Timestamp.from(FixedTimeSource.now))
     ps.executeUpdate()
   }
+
+  //scalastyle:off magic.number
```

Comment:
Can you switch magic number detection on again at the end of the test? Otherwise it remains off - I don't know whether it's for the rest of the file or the rest of the scan.

Reply:
Ah, I see; I didn't know this. Thanks!